src/02-crm/contacts.apib

```diff
@@ -177,6 +177,7 @@ Add a new contact.
         + tags: prospect, expo (array[string], optional)
         + custom_fields (array[CustomFieldValue], optional)
         + marketing_mails_consent: false (boolean, optional)
+        + `national_identification_number`: BE 3478 (string)
```

Comment:
PP: it would be better to move this property somewhere after IBAN and BIC, before the more "preferency" or metadata properties.